src/transformers/models/maskformer/feature_extraction_maskformer.py

```diff
@@ -64,8 +68,12 @@ class MaskFormerFeatureExtractor(FeatureExtractionMixin, ImageFeatureExtractionM
         image_std (`int`, *optional*, defaults to `[0.229, 0.224, 0.225]`):
             The sequence of standard deviations for each channel, to be used when normalizing images. Defaults to the
             ImageNet std.
-        ignore_index (`int`, *optional*, default to 255):
-            Value of the index (label) to ignore.
+        ignore_index (`int`, *optional*):
+            Value of the index (label) to ignore in the loss function.
```

Comment:
In SegFormer, users can specify the ignore index of the loss function with `model.config.semantic_loss_ignore_index` (and it's set to 255 by default). 

However, for MaskFormer, can users set the ignore index of the loss function of the model? I don't find any ignore index being set in `modeling_maskformer.py`, hence -100 is used by default, I suppose, and it can't be set to 255.

Reply:
The loss in maskformer doesn't use the `ignore_index`

Reply:
Yeah, so this argument is not useful in that case right?

Reply:
it is used in the feature extractor